java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -28,46 +46,274 @@ public void setUp() {
+
+        // exercise
+        CompletableFuture<String> response = service.info();
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(testPayload, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void info_with_multiple_InfoOptions_returns_success() {
+        // setup
+        String[] arguments =
+                new String[] {InfoOptions.Section.ALL.toString(), InfoOptions.Section.DEFAULT.toString()};
+        CompletableFuture<String> testResponse = mock(CompletableFuture.class);
+        String testPayload = "Key: Value";
+        when(testResponse.get()).thenReturn(testPayload);
+        when(commandManager.<String>submitNewCommand(eq(INFO), eq(arguments), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        InfoOptions options =
+                InfoOptions.builder()
+                        .section(InfoOptions.Section.ALL)
+                        .section(InfoOptions.Section.DEFAULT)
+                        .build();
+        CompletableFuture<String> response = service.info(options);
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(testPayload, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void info_with_empty_InfoOptions_returns_success() {
+        // setup
+        CompletableFuture<String> testResponse = mock(CompletableFuture.class);
+        String testPayload = "Key: Value";
+        when(testResponse.get()).thenReturn(testPayload);
+        when(commandManager.<String>submitNewCommand(eq(INFO), eq(new String[0]), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String> response = service.info(InfoOptions.builder().build());
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(testPayload, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void get_returns_success() {
+        // setup
+        String key = "testKey";
+        String value = "testValue";
+        CompletableFuture<String> testResponse = mock(CompletableFuture.class);
+        when(testResponse.get()).thenReturn(value);
+        when(commandManager.<String>submitNewCommand(eq(GET_STRING), eq(new String[] {key}), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String> response = service.get(key);
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(value, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void set_returns_success() {
+        // setup
+        String key = "testKey";
+        String value = "testValue";
+        CompletableFuture<Void> testResponse = mock(CompletableFuture.class);
+        when(testResponse.get()).thenReturn(null);
```

Comment:
This line is no-op. A mock always returns null for all calls unless reconfigured.
but null != void
```suggestion
        when(testResponse.get()).thenReturn(Void);
```
Is it possible?